lms/views/error.py

```diff
@@ -48,3 +49,19 @@ def error(self):
         return {'message': _("Sorry, but something went wrong. "
                              "The issue has been reported and we'll try to "
                              "fix it.")}
+
+
+    @view_config(context=MissingLTILaunchParamError)
+    def missing_lti_launch_param_error(self):
+        """
+        Catch MissingLTILaunchParamError, render a 400 page and report the it to Sentry.
+
+        If code raises a MissingLTILaunchParamError it means that a required parameter
+        was missing from an LTI launch request that we received and lti was not
+        successfully launched:
```

Comment:
I'm not sure that "lti was not successfully launched" makes sense. LTI is the Learning Tools Interoperability standard, so this is like saying "HTTP was not successfully launched" or something. An "LTI launch request" is an LTI-compatible form submission that we receive from Canvas when it wants us to "launch" our app. So it could say "the app was not successfully launched" or it could just say "a required parameter was missing from an LTI launch request that we received." and leave it at that?